packages/core/content-releases/server/src/services/release.ts

```diff
@@ -220,6 +230,18 @@ const createReleaseService = ({ strapi }: { strapi: LoadedStrapi }) => ({
       data: releaseWithCreatorFields,
     });
 
+    if (strapi.features.future.isEnabled('contentReleasesScheduling')) {
+      const schedulingService = getService('scheduling', { strapi });
+
+      if (releaseData.scheduledAt) {
+        // set function always cancel the previous job if it exists, so we can call it directly
+        await schedulingService.set(id, releaseData.scheduledAt);
+      } else if (release.scheduledAt) {
+        // When user don't send a scheduledAt and we have one on the release, means that user want to unschedule it
+        schedulingService.cancel(id);
+      }
```

Comment:
if it was already scheduled, is the previous scheduling removed in `schedulingService.set` ?

Reply:
Yes, `schedulingService.set` always try to cancel the previous job if it exists